modules/nodes-descheduler-about.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-descheduler.adoc
+
+[id="nodes-descheduler-about_{context}"]
+= About the descheduler
+
+You can use the descheduler to evict Pods based on specific strategies so that the Pods can be rescheduled onto more appropriate nodes.
+
+You can benefit from descheduling already-running Pods in situations such as the following:
```

Comment:
The word "already-running" threw me off. I suggest one of the following:
_...from descheduling Pods already running in situations..._
or
_...from descheduling running Pods in situations..._

On the other hand, if this is a common term that's just new to me, I'd leave it.

Reply:
Nah it was taken in a carryover from 3.11 docs. Agree to just saying running.